### PR TITLE
docs: update install guidance

### PR DESCRIPTION
**Pull request by EDIflyer**
I hadn't realised the docs weren't in all branches so previous PR had been off the default staging. Have now cherry-picked my changes across to a branch of `feat-docs` instead, hopefully this will only show those commits.

### CODE REVIEW

File: docs/docs/self-host/docker/run-locally.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Self-host OpenSign™ with Docker on Linux/MacOS/Windows
 import ReactPlayer from 'react-player';
 
 ## Steps to run OpenSign™ with docker on Linux/MacOS/Windows:
-  1. Firstly, install [Docker](https://www.docker.com/products/docker-desktop/) and ensure it is running before proceeding to the next steps. Make sure that you have [NodeJS](https://nodejs.org/en) & [GIT](https://git-scm.com/downloads) installed on your machine.
```
Comment (Zathiel) [inline docs/docs/self-host/docker/run-locally.md:7]: why do you want to remove that as NPM is mandatory ?

**Review by Zathiel — APPROVED** [review]
looks good to me

**Review by Zathiel — APPROVED** [review]
looks good to me

File: docs/docs/self-host/docker/run-locally.md (hunk 1 above)
Comment (EDIflyer) [reply docs/docs/self-host/docker/run-locally.md:7]: As far as I can tell you don't need to install NodeJS or Git if you're doing the Docker install as everything it contained within the container?

**Pull request merged**